### PR TITLE
Backport: Add test for runpath file

### PR DESCRIPTION
**Pull request by sondreso**
Backport of https://github.com/equinor/ert/pull/688

### CODE REVIEW

File: test-data/local/poly_example/assert_runpath_file.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        rpf = "".join(
+            [
+                runpath_line.format(iens=iens, iter=iteration, pwd=curdir)
+                for iens in [1, 2, 4, 8, 16, 32, 64]
+            ]
+        )
+        assert fh.read() == rpf
+
+    with open(ok_filename.format(iteration), "w") as fh:
+        fh.write(":)")
```
Comment (markusdregi) [inline test-data/local/poly_example/assert_runpath_file.py]: :)

Comment (sondreso) [reply test-data/local/poly_example/assert_runpath_file.py]: :slightly_smiling_face:

File: test-data/local/poly_example/assert_runpath_file.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+import sys
+import os
+
+
+def run():
```
Comment (markusdregi) [inline test-data/local/poly_example/assert_runpath_file.py]: Should we do `run(runpath_file, curdir)` and then `run(sys.argv[1], sys.argv[2])`?

Comment (markusdregi) [inline test-data/local/poly_example/assert_runpath_file.py]: Should we do `run(runpath_file, curdir)` and then `run(sys.argv[1], sys.argv[2])`?

**Pull request merged**